src/mmap.rs

```diff
@@ -288,13 +312,18 @@ impl Bytes<MemoryRegionAddress> for GuestRegionMmap {
     /// ```
     /// # extern crate vmm_sys_util;
     /// # use vmm_sys_util::tempfile::TempFile;
-    /// # use vm_memory::{Address, Bytes, GuestAddress, GuestMemoryMmap};
-    /// # use std::fs::OpenOptions;
-    /// # let start_addr = GuestAddress(0x1000);
-    /// # let gm = GuestMemoryMmap::from_ranges(&vec![(start_addr, 0x400)]).unwrap();
-    ///   let mut file = TempFile::new().unwrap().into_file();
-    ///   let mut mem = [0u8; 1024];
-    ///   gm.write_to(start_addr, &mut file, 128).unwrap();
+    /// use vm_memory::{Address, Bytes, GuestAddress, GuestMemoryMmap};
+    ///
+    /// let start_addr = GuestAddress(0x1000);
+    /// let gm = GuestMemoryMmap::from_ranges(&vec![(start_addr, 0x400)])
+    ///     .expect("Could not create guest memory");
+    /// # let mut file = TempFile::new().unwrap().into_file();
+    /// # /*
+    /// let mut file = /* temp file */;
+    /// # */
```

Comment:
This is interesting, but somehow makes reading the code example unnecessary hard. Do you have a strong preference for a generated example that doesn't use `TempFile`?

Reply:
`TempFile` seemed like just a random test helper class, referencing it seemed more confusing than helpful.

Reply:
I've aligned this and with the other `write_to` examples in `guest_memory.rs`. Now both use `/dev/null` in the visible example and use `TempFile` as a fallback for non `cfg!(unix)` environments.

Thoughts?